keps/sig-api-machinery/20180415-crds-to-ga.md

```diff
@@ -162,10 +163,65 @@ TODO: complete this list
 
 ### Scale Targets for GA
 
-* TODO quantify: Read/write latency of CRDs within X% of native Kubernetes types
-* TODO quantify: Latency degrades less than X% for up to 100k Custom Resources per CRD kind
-* TODO quantify: Webhook conversion QPS of a noop converter is within X% of QPS with no webhook
-* Coordinate with sig-scalability
+The scale target for GA of custom resources are defined by the same [API call latency
+SLIs/SLOs as the Kuberetes native types](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/api_call_latency.md#api-call-latency-slisslos-details).
+
+The targets are defined by the below thresholds, which are organized the same way as the [Kubernetes native type thresholds](https://github.com/kubernetes/community/blob/master/sig-scalability/configs-and-limits/thresholds.md#kubernetes-thresholds), with only a couple changes:
+
+- Since custom resources can be arbitrarily large, we have added a object size column and thresholds for a range of object sizes.
+- For clarity, we include thersolds with and without conversion webhooks.
+
+**Custom Resources per Definition, with Conversion webhook:**
+
+| Object size | Threshold scope=namespace | Threshold: scope=cluster |
+| --- | --- | --- |
+| 10kb | 2500 | 12500 |
+| 25kb | 1000 | 5000 |
+| 50kb | 500 | 2500 |
+
+**Custom Resources per Definition, without Conversion webhook:**
```

Comment:
Why should conversion webhook or not have a bearing on the number of objects supported? I can see it making a difference for the size of objects or latency, but why the count?

Reply:
The Kubernetes [SLI/SLOs are defined uniformly across all types](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/api_call_latency.md#api-call-latency-slisslos-details), where the suggested maximum count (or "threshold") of each type is found to meets the SLI/SLOs.

As part of demoting the "without conversion webhook" numbers, I hope this will be a bit less confusing.